lib/shared/types.js

```diff
@@ -152,7 +152,7 @@ module.exports = {};
 /**
  * @typedef {Object} DeprecatedInfo
  * @property {string} [message] General message presented to the user
- * @property {string} [url] URL to more information about this deprecation in general
+ * @property {?string} [url] URL to more information about this deprecation in general
```

Comment:
```suggestion
 * @property {string} [url] URL to more information about this deprecation in general
```

Actually `url` isn't nullable: https://github.com/eslint/rewrite/blob/core-v0.13.0/packages/core/src/types.ts#L214-L217

Reply:
Oh - I think this is just me not being familiar with JSDoc type syntax - i'd have assumed `?string` meant `string | undefined` or "this is optional", the way that `a?: string` means roughly "`a` is optional"/"`a` has type `string | undefined`" in TS. If I understand correctly, though, `url` being optional is already expressed by the `[url]` syntax, is that right?